target/linux/qualcommax/ipq807x/base-files/lib/preinit/09_mount_factory_data

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+preinit_mount_factory_data() {
+	local mtd_path
+
+	. /lib/functions.sh
+	. /lib/functions/system.sh
+
+	case $(board_name) in
+	tplink,eap660hd-v1)
+		mtd_path=$(find_mtd_chardev "factory_data")
+		ubiattach --dev-path="$mtd_path" --devn=1
+		mkdir /tmp/factory_data
+		mount -o ro,noatime -t ubifs ubi1:ubi_factory_data /tmp/factory_data
```

Comment:
Can you share the file structure that this FS contains?

Reply:
```
-rw-rw-r--    1 500      500              4 Nov 20  2020 add-hardver
-r-sr-Sr-x    1 root     root             6 Jan  1  2020 default-mac
--wS--sr-x    1 root     root            40 Jan  1  2020 device-id
-rwxr-xr-x    1 root     root            74 Jan  1 00:00 device-info
--ws--x--t    1 root     root        131072 Jan  1  2020 radio
-r-sr-Sr-x    1 root     root             8 Jan  1  2020 region
```